sourced/ml/models/df.py

```diff
@@ -58,6 +58,13 @@ def docs(self) -> int:
         """
         return self._docs
 
+    @property
+    def df(self) -> dict:
```

Comment:
Now I see. We need to put a huge comment which restricts from exposing the dictionary directly.

Reply:
okay, adding